ui/raidboss/data/05-shb/raid/e1s.js

```diff
@@ -492,273 +492,274 @@
+        'Regained Thunder III': 'リゲイン・サンダガ',
+        'Spear of Paradise': 'スピア・オブ・パラダイス',
+        'Vice and Virtue': 'ヴァイス・アンド・ヴァーチュー',
+        'Vice of Apathy': 'ヴァイス・オブ・アパシー',
+        'Vice of Greed': 'ヴァイス・オブ・グリード',
+        'Vice of Pride': 'ヴァイス・オブ・プライド',
+        'Vice of Sloth': 'ヴァイス・オブ・スロース',
+        'Vice of Thievery': 'ヴァイス・オブ・シーヴァリィ',
+        'Vice of Vanity': 'ヴァイス・オブ・ヴァニティー',
       },
       '~effectNames': {
-        'Slippery Prey': 'マーキング対象外',
-        'Prey': 'マーキング',
-        'Poison': '毒',
-        'Physical Vulnerability Up': '被物理ダメージ増加',
-        'Magic Vulnerability Up': '被魔法ダメージ増加',
-        'Lightning Resistance Down II': '雷属性耐性低下［強］',
-        'Healing Magic Down': '回復魔法効果低下',
-        'Fetters': '拘束',
         'Bleeding': 'ペイン',
+        'Fetters': '拘束',
+        'Healing Magic Down': '回復魔法効果低下',
+        'Lightning Resistance Down II': '雷属性耐性低下［強］',
+        'Magic Vulnerability Up': '被魔法ダメージ増加',
+        'Paradise Regained': 'パラダイスリゲイン',
+        'Physical Vulnerability Up': '被物理ダメージ増加',
+        'Poison': '毒',
+        'Prey': 'マーキング',
+        'Slippery Prey': 'マーキング対象外',
       },
     },
     {
       'locale': 'cn',
       'replaceSync': {
-        'Eden Prime': '至尊伊甸',
         'Arcane Sphere': '立体魔法阵',
+        'Eden Prime': '至尊伊甸',
         'Guardian of Paradise': '伊甸守护者',
       },
       'replaceText': {
+        'Delta Attack (Cross)': '三角攻击(角落)',
+        'Delta Attack (Donut)': '三角攻击(月环)',
+        'Delta Attack': '三角攻击',
+        'Dimensional Shift': '空间转换',
+        'Eden\'s Blizzard III': '伊甸冰封',
+        'Eden\'s Fire III': '伊甸爆炎',
+        'Eden\'s Flare': '伊甸核爆',
+        'Eden\'s Gravity': '伊甸重力',
+        'Eden\'s Thunder III': '伊甸暴雷',
+        'Eternal Breath': '永恒吐息',
+        'Fragor Maximus': '极大爆炸',
+        'Heavensunder': '天国分断',
+        'Mana Boost': '魔力增幅',
+        'Mana Burst': '魔力爆发',
+        'Mana Slice': '魔力斩击',
+        'Paradisal Dive': '乐园冲',
+        'Paradise Lost': '失乐园',
+        'Paradise Regained': '复乐园',
+        'Primeval Stasis': '原初停滞',
+        'Pure Beam': '净土射线',
+        'Pure Light': '净土之光',
+        'Regained Blizzard III': '复乐园冰封',
+        'Regained Fire III': '复乐园爆炎',
+        'Regained Thunder III': '复乐园暴雷',
+        'Spear [oO]f Paradise': '乐园之枪',
         'Vice [oO]f Vanity': '虚荣之恶',
-        'Vice of Thievery': '盗窃之恶',
-        'Vice of Sloth': '怠惰之恶',
-        'Vice of Pride': '傲慢之恶',
-        'Vice of Greed': '贪婪之恶',
-        'Vice of Apathy': '冷漠之恶',
-        'Vice And Virtue! (T)': '恶习与美德！(坦克)',
-        'Vice And Virtue! (D)': '恶习与美德！(DPS)',
-        'Vice And Virtue! (H)': '恶习与美德！(奶妈)',
-        'Vice And Virtue (T)': '恶习与美德(坦克)',
         'Vice And Virtue (D)': '恶习与美德(DPS)',
         'Vice And Virtue (H)': '恶习与美德(奶妈)',
+        'Vice And Virtue (T)': '恶习与美德(坦克)',
+        'Vice And Virtue! (D)': '恶习与美德！(DPS)',
+        'Vice And Virtue! (H)': '恶习与美德！(奶妈)',
+        'Vice And Virtue! (T)': '恶习与美德！(坦克)',
```

Comment:
I think there are 3 doubles. Which ones do you want to keep?

Reply:
I'm not sure what you mean about doubles.  The exclamation point is there to (subtly?) indicate that it's the powered up version vs the normal version.

Reply:
Ah I understand. It's like French, except that I just put the punctuation aside, I didn't have to create more. I did not think about it. thank you for enlightening me.